clarity_ext/domain/aliquot.py

```diff
@@ -4,9 +4,9 @@
 
 class Aliquot(Artifact):
     """
-    NOTE: This class currently acts as a base class for both Analyte and ResultFile. It will be
-    merged with Analyte, since the name can cause some confusion as Analytes
-    and ResultFiles strictly don't need to be Aliquots, i.e. they can be non-divided copies
+    NOTE: This class currently acts as a base class for both Analyte and ResultFile. It should
+    be renamed (e.g. AnalyteBase), since the name can cause some confusion as Analytes
```

Comment:
So why don't you just change the name?

Reply:
Good point :) I was just fixing this comment, because I thought the comment (which I think I wrote) was wrong. I.e. I thought this class shouldn't be merged with Analyte. But now when I'm thinking about it, should it maybe? I think it would be good to discuss this at some point. I'm often confused about the distinction between Analyte and Aliquot in clarity-ext.

I did however not want to change the name in this commit because we're just hotfixing. In any case, I think I'll revert to the old text.